src/main/java/com/example/tour/activity/ActivityController.java

```diff
@@ -9,7 +9,7 @@
 @RestController
 @RequestMapping("api/activity")
 public class ActivityController {
-    private ActivityService activityService;
+    private final ActivityService activityService;
```

Comment:
Does this work, with the "final" on it?

Reply:
This works actually,
I think this is the recommended one,